app/components/address-search.hbs

```diff
@@ -1,65 +1,48 @@
-      <:error>{{! Disable the default error message}}</:error>
-    </Item>
-    <Item
-      @labelFor="{{id}}-address-search-bus-select"
-      @required={{not this.isBusNumberSelectionDisabled}}
-    >
-      <:label>Busnummer</:label>
-      <:content>
-        <AddressRegisterBusNumberSelector
-          @id="{{id}}-address-search-bus-select"
-          @address={{this.addressWithBusNumber}}
-          @options={{this.addressesWithBusNumbers}}
-          @onChange={{this.handleBusNumberChange}}
-          @disabled={{this.isBusNumberSelectionDisabled}}
-          @placeholder={{if
-            this.showBusNumbersNotAvailableMessage
-            "Geen busnummer beschikbaar"
-          }}
-        />
+{{#if this.isAddressSearchMode}}
+<Item @labelFor="{{id}}-address-search" @required={{@required}} @errorMessage={{if @error "Selecteer een adres" }}
+  @alignTop={{true}}>
+  <:label>Adres</:label>
+  <:content as |errorMessage|>
+    <AddressRegisterSelector @id="{{id}}-address-search" @address={{@address}} @onChange={{this.handleAddressChange}}
+      @error={{errorMessage}} />
+    {{#if errorMessage}}
+    <AuHelpText @error={{true}}>{{errorMessage}}</AuHelpText>
+    {{/if}}
+    <p class="au-u-margin-top-tiny">
+      <AuButton @skin="link" {{on "click" this.toggleInputMode}}>
+        Vul adres manueel in
+      </AuButton>
+    </p>
+  </:content>
+  <:error>{{! Disable the default error message}}</:error>
+</Item>
+<Item @labelFor="{{id}}-address-search-bus-select" @required={{not this.isBusNumberSelectionDisabled}}>
+  <:label>Busnummer</:label>
+  <:content>
+    <AddressRegisterBusNumberSelector @id="{{id}}-address-search-bus-select" @address={{this.addressWithBusNumber}}
+      @options={{this.addressesWithBusNumbers}} @onChange={{this.handleBusNumberChange}}
+      @disabled={{this.isBusNumberSelectionDisabled}} @placeholder={{if
+      this.showBusNumbersNotAvailableMessage "Geen busnummer beschikbaar" }} />
 
-        {{#if this.selectedAddress}}
-          <AuHelpText>
-            Staat het correcte busnummer niet in de lijst?
-            <AuLinkExternal href="mailto:{{config "contactEmail"}}">Mail ons</AuLinkExternal>
-          </AuHelpText>
-        {{/if}}
-      </:content>
-    </Item>
-  {{else}}
-    {{yield to="manual-address-input"}}
+    {{#if this.selectedAddress}}
+    <AuHelpText>
+      Staat het correcte busnummer niet in de lijst?
+      <AuLinkExternal href="mailto:{{config " contactEmail"}}">Mail ons</AuLinkExternal>
+    </AuHelpText>
+    {{/if}}
+  </:content>
+</Item>
+{{else}}
+{{yield to="manual-address-input"}}
 
-    <Item>
-      <:content>
-        <AuButton @skin="link" {{on "click" this.toggleInputMode}}>
-          Vind adres in lijst
-        </AuButton>
-      </:content>
-    </Item>
-  {{/if}}
+<Item>
+  <:content>
+    <AuButton @skin="link" {{on "click" this.toggleInputMode}}>
+      Vind adres in lijst
+    </AuButton>
+  </:content>
+</Item>
+{{/if}}
+{{yield to="common-input"}}
```

Comment:
I would move this above the `Vind adres in lijst` link maybe? I think that will look nicer. It should be possible to yield the block from multiple places so that the item appears in both "modes".